benchmark/src/main/content/bin/kc-chaos.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Randomly delete pods in a Kubernetes namespace.
+set -e
+
+: ${INITIAL_DELAY_SECS:=30}
+: ${CHAOS_DELAY_SECS:=60}
+: ${PROJECT:="runner-keycloak"}
+
+sleep $INITIAL_DELAY_SECS
+echo -e "\033[0;31m INFO: STARTING CHAOS SCENARIO - DELETING RANDOM KC POD with a CHAOS_DELAY_SECS of $CHAOS_DELAY_SECS"
+while true; do
+  RANDOM_KC_POD=$(kubectl \
+    -n "${PROJECT}" \
+    -o 'jsonpath={.items[*].metadata.name}' \
+    get pods -l app=keycloak | \
+      tr " " "\n" | \
+      shuf | \
+      head -n 1)
+  kubectl exec -n "${PROJECT}" "${RANDOM_KC_POD}" -- /bin/sh -c "kill 1"
```

Comment:
I thought `kill 1` didn't work. Isn't the keycloak pod immediately restarted?

Reply:
@pruivo the `kill -9 1` doesn't work on my end, but `kill 1` works and restarts the pod immediately

Reply:
going to use the suggestion from #363. closing this thread.